src/Extensions/src/App.Metrics.Health.Extensions.DependencyInjection/App.Metrics.Health.Extensions.DependencyInjection.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="App.Metrics.Health" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
```

Comment:
What behavior (outcome) you expect without version here?

Reply:
And one bellow

Reply:
Same here - versions updated globally in top-level `Directory.Build.targets`